tests/Entity/ProfileTest.php

```diff
@@ -50,4 +67,91 @@ public function testGetAttributeName()
         $phoneNumber = $this->profile->getPhoneNumber();
         $this->assertEquals('phone_number', $phoneNumber->getName());
     }
+
+    public function testGetAgeVerifications()
+    {
+        $profileData = [
+            'age_over:18' => new Attribute('age_over:18', 'true', [], []),
+            'age_under:18' => new Attribute('age_under:18', 'false', [], []),
+        ];
+        $processor = new Processor($profileData);
```

Comment:
Looks like we are testing _Processor_ rather than _Profile_.  This should be in its own test class.

Reply:
It does look like, because we don't have a sample receipt with age derivations.
I am happy to move it inside ProcessorTest.